collect_app/src/test/java/org/odk/collect/android/external/FormUriActivityTest.kt

```diff
@@ -1144,6 +1144,37 @@ class FormUriActivityTest {
         )
     }
 
+    @Test
+    fun `When attempting to view a non-editable form that uses entities and the forms database is locked then start form for view only`() {
```

Comment:
Would this test not be enough? We'd maybe want to add an assertion that here that opening the view-only form doesn't lock the forms change lock, but I'm not sure if we need the two connected tests?

Reply:
I have added one extra test here and removed those connected ones.